internal/constants.go

```diff
@@ -3,5 +3,5 @@ package internal
 const (
 	// JSONSchemaVersion is the current schema version output by the JSON encoder
 	// This is roughly following the "SchemaVer" guidelines for versioning the JSON schema. Please see schema/json/README.md for details on how to increment.
-	JSONSchemaVersion = "14.0.0"
+	JSONSchemaVersion = "14.0.1"
```

Comment:
Is this "MINOR when you add backwards-compatible functionalities" instead of "PATCH when you make backwards-compatible bug fixes"?

Reply:
This isn't following [semver](https://semver.org/), but instead [schemaver](https://docs.snowplow.io/docs/pipeline-components-and-applications/iglu/common-architecture/schemaver/) https://github.com/anchore/syft/blob/main/schema/json/README.md#versioning

> Given a version number format MODEL.REVISION.ADDITION:
> - MODEL: increment when you make a breaking schema change which will prevent interaction with any historical data
> - REVISION: increment when you make a schema change which may prevent interaction with some historical data
> - ADDITION: increment when you make a schema change that is compatible with all historical data